avm/res/network/route-table/main.bicep

```diff
@@ -0,0 +1,134 @@
+metadata name = 'Route Tables'
```

Comment:
Please also make sure of the following:
- update codeowners file uncommenting the line referencing your module
- make sure your module owner team is properly added to the intended parent team (cc @matebarabas)

Reply:
Done, also created both teams

https://github.com/orgs/Azure/teams/avm-res-network-routetable-module-owners-bicep 
https://github.com/orgs/Azure/teams/avm-res-network-routetable-module-contributors-bicep

and added corresponding parent teams as per https://azure.github.io/Azure-Verified-Modules/specs/shared/#grant-permissions---bicep